### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by v-zhuravlev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,32 +128,110 @@ and some optional params can be provided as well:
     modbus_read[{$MODBUS_PORT},32,4,3,uint32,BE,0]
     modbus_read[enc://192.168.1.1,1,6,1]
     modbus_read[tcp://192.168.1.1:5000,1,5,1]
+    modbus_read[{$MODBUS_PORT},{$MODBUS_SLAVE},10,1,100*bit]
+    modbus_read[{$MODBUS_PORT},{$MODBUS_SLAVE},10,1,10*double]
+    modbus_read[{$MODBUS_PORT},{$MODBUS_SLAVE},10,3,2*uint16+skip+float]
+    modbus_read[{$MODBUS_PORT},{$MODBUS_SLAVE},10,3,2*uint16+double+float*3,MLE]
```
Comment (i-ky) [inline README.md]: Let's show how `BE` and `LE` can be used here!

File: README.md [hunk 2]
```diff
@@ -132,32 +128,110 @@ and some optional params can be provided as well:
...
+
+### 5.1. The Simplest Example
+The simplest datatype expression would be 
+`2*uint16` that you write in the key: `modbus_read[{$MODBUS_PORT},{$MODBUS_SLAVE},10,3,2*uint16]`  
+This expression is quite simple - it just retrieves two unsigned integers starting from register 10. Here is what you would get:
+```
+{
+  "10":123,
+  "11":321
+}
```
Comment (i-ky) [inline README.md]: Actually, the result won't be pretty-printed like that. Should we mention it?

File: README.md [hunk 3]
```diff
@@ -132,32 +128,110 @@ and some optional params can be provided as well:
...
+- Name: Modbus bulk request
+- Key: `modbus_read[{$MODBUS_PORT},{$MODBUS_SLAVE},10,3,2*uint16]`  
+- Type: Simple check or Zabbix agent
+- Type of information: Text  
+- History: 0 (recommended)  
+![image](https://user-images.githubusercontent.com/14870891/45901178-c3d03800-bdea-11e8-811e-f2fcee702beb.png)
+
+
+
+Then, create two [dependent items](https://www.zabbix.com/documentation/3.4/manual/config/items/itemtypes/dependent_items):
```
Comment (i-ky) [inline README.md]: You seem to mix links to 3.4 and 4.0 documentation. Let's be consistent.

File: README.md [hunk 4]
```diff
@@ -132,32 +128,110 @@ and some optional params can be provided as well:
...
-Testing Modbus connectivity is easy with *modpoll* command utility.  
-You may try to grab some modbus registers with it before you try to do it with Zabbix.  
-http://www.modbusdriver.com/modpoll.html
+Testing Modbus connectivity is easy with [*modpoll*](http://www.modbusdriver.com/modpoll.html) command utility.  
+You may try to grab some Modbus registers with it before you try to do it with Zabbix.  
 
-If using libzbxmodbus with Zabbix agent, you can also save time and test responses first with `zabbix_get`, for example:
+
+If you use libzbxmodbus with Zabbix agent, then you can also save time and test responses first with `zabbix_get`, for example:
 `zabbix_get -s localhost -kmodbus_read["/dev/ttyS1 9600 N",9,0x1518,3,l,1,0]`
```
Comment (i-ky) [inline README.md]: I have a habit of always wrapping key in single quotes:
```
$ zabbix get -s localhost -k 'modbus_read[...]'
```
I think it is a good habit, saves from special character issues in 90% of situations.

File: README.md [hunk 5]
```diff
@@ -132,32 +128,110 @@ and some optional params can be provided as well:
...
   
-## 5. Known Issues and Limitations  
+## 7. Known Issues and Limitations
```
Comment (i-ky) [inline README.md]: It seems there is a [way](https://support.typora.io/Auto-Numbering/) to make headings auto-numbered.

File: README.md [hunk 6]
```diff
@@ -104,21 +96,25 @@ and some optional params can be provided as well:
       `d` or `double`- for MODBUS_FLOAT64, 64bit  
     
     otherwise, defaults will be used:  
-      MODBUS_BIT if modbus function 1 or 2.  
-      MODBUS_UINT16 if modbus_function 3 or 4.  
+      MODBUS_BIT if modbus_function is `1` or `2`.  
+      MODBUS_UINT16 if modbus_function is `3` or `4`.  
+
+    Note: Datatypes can be combined in the datatype expression here to request more than one register at once. See section (5) below for how to do this.
```
Comment (i-ky) [inline README.md]: Anchor link (`#heading-title`) can be used here.

File: README.md (hunk 2 above)
Comment (v-zhuravlev) [reply README.md]: That's ok,  Here it's easier to read like that.

**Pull request merged**